app/Filament/Admin/Resources/Users/UserResource.php

```diff
@@ -119,44 +144,318 @@ public static function defaultTable(Table $table): Table
+                            ->columns([
+                                'default' => 1,
+                                'md' => 3,
+                                'lg' => 3,
+                            ])
+                            ->schema([
+                                TextInput::make('username')
+                                    ->label(trans('admin/user.username'))
+                                    ->columnSpan([
+                                        'default' => 1,
+                                        'md' => 1,
+                                        'lg' => 1,
+                                    ])
+                                    ->required()
+                                    ->unique()
+                                    ->maxLength(255),
+                                TextInput::make('email')
+                                    ->label(trans('admin/user.email'))
+                                    ->columnSpan([
+                                        'default' => 1,
+                                        'md' => 1,
+                                        'lg' => 1,
+                                    ])
+                                    ->email()
+                                    ->required()
+                                    ->unique()
+                                    ->maxLength(255)
+                                    ->hintAction(
+                                        Action::make('test')
+                                            ->label(trans('admin/setting.mail.test_mail'))
+                                            ->icon('tabler-send')
+                                            ->hidden(fn () => config('mail.default', 'log') === 'log')
+                                            ->action(function (User $user) {
+                                                try {
+                                                    MailNotification::route('mail', $user->email)
+                                                        ->notify(new MailTested($user));
+
+                                                    Notification::make()
+                                                        ->title(trans('admin/setting.mail.test_mail_sent'))
+                                                        ->success()
+                                                        ->send();
+                                                } catch (Exception $exception) {
+                                                    Notification::make()
+                                                        ->title(trans('admin/setting.mail.test_mail_failed'))
+                                                        ->body($exception->getMessage())
+                                                        ->danger()
+                                                        ->send();
+                                                }
+                                            })
```

Comment:
What's the point of sending a test mail to the user?

Reply:
Valid... Removed